ibm/service/database/resource_ibm_database.go

```diff
@@ -2401,7 +2340,10 @@ func waitForDatabaseTaskComplete(taskId string, d *schema.ResourceData, meta int
 	delayDuration := 5 * time.Second
 
 	timeout := time.After(t)
-	delay := time.Tick(delayDuration)
+	ticker := time.NewTicker(delayDuration)
+	delay := ticker.C
+	defer ticker.Stop()
```

Comment:
Why are we changing the delay logic?

Reply:
This got left in while debugging `waitForDatabaseTaskComplete` but allows the Ticker to be garbage collected see https://pkg.go.dev/time#Tick